hoomd/mpcd/CollisionMethod.cc

```diff
@@ -395,6 +420,69 @@ void mpcd::CollisionMethod::transferRigidBodyMomenta(uint64_t timestep)
             {
             angmom_change_body.z = Scalar(0);
             }
+
+        // calculate scaling factor
+        Scalar a = 0;
+        Scalar b = 0;
+        Scalar c = 0;
+        Scalar s = 0;
+
+        const quat<Scalar> initial_angmom_body = Scalar(0.5) * conj(orientation) * angmom;
+        if (inertia.x > Scalar(0))
```

Comment:
```suggestion
        if (inertia.x != Scalar(0))
```

Reply:
Often in the hoomd code, when inertia is checked to not be zero, they use greater than such as in the ThermoCompute when the rotational kinetic energy is calculated. Going by that example, it would make just as much sense to replace `Scalar(0)` with `0`